templates/zerver/help/index.md

```diff
@@ -91,7 +89,7 @@ as an **organization**.
 * [Rename a stream](/help/rename-a-stream)
 * Preview a stream (not implemented)
 * [Unsubscribe from a stream](/help/unsubscribe-from-a-stream)
-* [Change who can join a stream](/help/change-who-can-join-a-stream)
+* [Change the accessibility of a stream](/help/change-the-accessibility-of-a-stream)
```

Comment:
I'm not sure it should be `accessibility` - I think it's used in the context of making sure the app is accessible to people with disabilities (e.g. blind) or accessibility of a service.

I don't think we should change it.